cmd/kubeadm/app/cmd/token.go

```diff
@@ -158,11 +158,11 @@ func NewCmdToken(out io.Writer, errW io.Writer) *cobra.Command {
 	tokenCmd.AddCommand(listCmd)
 
 	deleteCmd := &cobra.Command{
-		Use:                   "delete [token-value]",
+		Use:                   "delete token-value ...",
```

Comment:
I'd prefer to the old way. Better not changing the style we're using in current kube command lines.

Reply:
And this usage is misleading. Users may think `token-value` is another new subcommand. But actually it is not.

Reply:
Well, I'd say that current syntax is misleading. It suggests that token-value is optional as [it's in square brackets](https://en.wikipedia.org/wiki/Usage_message#Pattern).

Anyway, how do you want it to be? ```delete [token-value] ...``` ?

Reply:
+1 for @dixudx 's comment. For better or worse, that's our current way of doing things in help. So we have to stick to it for now and, possibly, change things in another PR.

Reply:
ok, makes sense to me. will chang it to ```delete [token-value] ...```